src/components/SkeletonLoader/SkeletonLoader.tsx

```diff
@@ -1,7 +1,119 @@
-import React from "react";
+import * as React from "react";
+import bem from "../../utils/bem";
+import {
+  SkeletonLoaderImageRatios,
+  SkeletonLoaderLayouts,
+} from "./SkeletonLoaderTypes";
 
-function SkeletonLoader() {
-  return <div className="skeleton-loader"></div>;
+export interface SkeletonLoaderProps {
+  /** Additional attributes passed to the horizontal rule */
```

Comment:
Copy/paste issue? When would this prop ever be extended and for what values? It'll be good to have an example of this. Otherwise, I think you can remove `attributes` as a prop and add the `width` value directly to the parent `div`.

Reply:
Ha.  Yes, that was pasted in and not changed over.  Oops.  Regardless, thanks for the suggestion.  I removed `attributes` altogether.